packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py

```diff
@@ -65,6 +67,7 @@ async def stop_dynamic_service(
         node_id=node_id,
         simcore_user_agent=simcore_user_agent,
         save_state=save_state,
+        primary_group_id=primary_group_id,
```

Comment:
side-comment: I think parsing the name of the method (line 66) every single time this gets called is an overkill. either create a constant and parse it only once, or just cast it, we know it passes the conventions.